project_event/models/project_project.py

```diff
@@ -104,14 +104,10 @@ def action_cancel(self):
 
     @api.multi
     def action_accept(self):
-        if self.state in ['draft', 'option', 'postponed', 'canceled']:
-            self.send_message('accepted')
-            return self.get_confirmation_wizard('accept')
+        return self.get_confirmation_wizard('accept')
```

Comment:
@eilst If this is the case, we don't require def action_accept(self): we can directly call with  self.get_confirmation_wizard('accept') . Correct me if i am wrong